### ISSUE TITLE
Release error

### ISSUE THREAD

**Issue by Cifer-Y** [bitwalker/exrm#151]
```
==> Building release with MIX_ENV=dev.
==> Generating relx configuration...
==> Generating sys.config...
==> Generating boot script...
** (ArgumentError) argument error
    (stdlib) :re.run([24050, 27979, 65, 80, 73, 46, 109, 100], {:re_pattern, 0, 0, 0, <<69, 82, 67, 80, 142, 0, 0, 0, 0, 0, 0, 0, 81, 0, 0, 0, 255, 255, 255, 255, 255, 255, 255, 255, 69, 0, 109, 0, 0, 0, 0, 0, 0, 0, 64, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...>>}, [capture: :none])
    lib/exrm/plugin.ex:92: ReleaseManager.Plugin.match_plugins/2
    (elixir) lib/enum.ex:1261: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/mix/tasks/release.ex:225: Mix.Tasks.Release.execute_before_hooks/1
    lib/mix/tasks/release.ex:69: Mix.Tasks.Release.do_run/1
    (mix) lib/mix/cli.ex:55: Mix.CLI.run_task/2
```

What is the reason? 
Thanks.

**Comment by bitwalker** [comment]
@Cifer-Y Could you provide more info on what version of exrm/Elixir/Erlang you are running, and whether you have any custom plugins?

**Comment by Cifer-Y** [comment]
Oh, yes, here

**Comment by bitwalker** [comment]
I have pushed a fix for this in 0.18.1.
